hibernate-envers/src/main/java/org/hibernate/envers/boot/EnversBootLogger.java

```diff
@@ -45,7 +45,7 @@ static Logger subLogger(String subName) {
 	@LogMessage( level = INFO )
 	@Message(
 			value = "Envers-generated HBM mapping...%n%s",
-			id = 90000001
+			id = 90005601
```

Comment:
This code change is to bypass a CI compilation error that the id doesn't fall within the id range. Seems an overlooking by Steve's recent merging.